src/app/core/services/sparque-categories/sparque-categories.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TestBed } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { anything, instance, mock, verify, when } from 'ts-mockito';
+
+import { SparqueCategoryTreeResponse } from 'ish-core/models/sparque-category-tree/sparque-category-tree.interface';
+import { SparqueCategoryTreeMapper } from 'ish-core/models/sparque-category-tree/sparque-category-tree.mapper';
+import { SparqueApiService } from 'ish-core/services/sparque-api/sparque-api.service';
+import { categoryTree } from 'ish-core/utils/dev/test-data-utils';
+
+import { SparqueCategoriesService } from './sparque-categories.service';
+
+describe('Sparque Categories Service', () => {
+  let sparqueApiService: SparqueApiService;
+  let sparqueCategoriesService: SparqueCategoriesService;
+  let sparqueCategoryTreeMapper: SparqueCategoryTreeMapper;
+  const apiVersion = 'v3';
+
+  beforeEach(() => {
+    sparqueApiService = mock(SparqueApiService);
+    sparqueCategoryTreeMapper = mock(SparqueCategoryTreeMapper);
+    when(sparqueApiService.get(anything(), apiVersion, anything())).thenReturn(
+      of<SparqueCategoryTreeResponse>({ categories: [] })
+    );
+    when(sparqueCategoryTreeMapper.fromData(anything())).thenReturn(categoryTree());
+
+    TestBed.configureTestingModule({
+      providers: [
+        { provide: SparqueApiService, useFactory: () => instance(sparqueApiService) },
+        { provide: SparqueCategoryTreeMapper, useFactory: () => instance(sparqueCategoryTreeMapper) },
+      ],
+    });
+    sparqueCategoriesService = TestBed.inject(SparqueCategoriesService);
+  });
+
+  it('should always delegate to sparque api service when called', () => {
```

Comment:
can you add a describe for testing the getTopLevelCategories ? - somehow it is not clear by the test descriptions